src/Capture/index.ts

```diff
@@ -16,18 +17,42 @@ export default class Capture {
     return this.permissionStatus = 'allowed'
   }
 
-  async camera(){
+  async camera(): Promise<Photo>{
     await this.checkPermissionStatus();
 
     const image = await Camera.getPhoto({
       quality: 100,
       allowEditing: true,
-      resultType: CameraResultType.Uri,
-      source: CameraSource.Prompt
+      resultType: CameraResultType.Base64,
+      source: CameraSource.Camera
     });
 
     console.log(image)
     return image
   }
 
+  async publish(images: Photo[]) {
+    const formData = new FormData();
+    images.forEach((image) => {
+      formData.append('image', image.base64String!);
```

Comment:
each image should have a different key, otherwise it will join all base64 together

Reply:
Fixed using element index